File: DMOJ/DMOPC '19 Contest 4 P0.rb
# frozen_string_literal: true

str1 = gets.to_s.chars
str2 = gets.to_s.chars
count = 0

(0...str1.length).each do |i|
  count += 1 if str1[i] != str2[i]
end
if count == 1
  puts("LARRY IS SAVED!")
else
  puts("LARRY IS DEAD!")
end

File: DMOJ/DMOPC '19 Contest 4 P0.py
str1 = list(input())
str2 = list(input())
count = 0

for i in range(len(str1)):
    if str1[i] != str2[i]:
        count += 1

if count == 1:
    print("LARRY IS SAVED!")
else:
    print("LARRY IS DEAD!")